sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/DataSourceUtils.scala

```diff
@@ -81,12 +81,16 @@ object DataSourceUtils extends PredicateHelper {
    * in a driver side.
    */
   def verifySchema(format: FileFormat, schema: StructType): Unit = {
+    checkFieldType(format, schema)
+    checkFieldNames(format, schema)
```

Comment:
Does this PR change anything? it looks like a refactoring by pulling out part of `verifySchema` as a separate method `checkFieldType`.

Reply:
> Does this PR change anything? it looks like a refactoring by pulling out part of `verifySchema` as a separate method `checkFieldType`.

oh, sorry, one file was not chosen when commit change